<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>1252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>831</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Slider和QColor</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Red</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3" rowspan="4">
              <widget class="QTextEdit" name="textEdit_2">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>170</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>170</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>240</red>
                     <green>240</green>
                     <blue>240</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Green</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Blue</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSlider" name="SliderBlue">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSlider" name="SliderGreen">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSlider" name="SliderRed">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="tracking">
                <bool>true</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
               <property name="tickPosition">
                <enum>QSlider::NoTicks</enum>
               </property>
               <property name="tickInterval">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>GroupBox</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="checkBox_1">
               <property name="text">
                <string>LED1</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_2">
               <property name="text">
                <string>LED2</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_3">
               <property name="text">
                <string>LED3</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_4">
               <property name="text">
                <string>LED4</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_5">
               <property name="text">
                <string>LED5</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_6">
               <property name="text">
                <string>LED6</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_7">
               <property name="text">
                <string>LED7</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_8">
               <property name="text">
                <string>LED8</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>shankeBox</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QTextEdit" name="textEdit_7"/>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>170</width>
                 <height>170</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>19</pointsize>
                </font>
               </property>
               <property name="smallDecimalPoint">
                <bool>false</bool>
               </property>
               <property name="digitCount">
                <number>2</number>
               </property>
               <property name="segmentStyle">
                <enum>QLCDNumber::Flat</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="textEdit_6"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>receiveBox</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QTextEdit" name="m_textEdit">
               <property name="maximumSize">
                <size>
                 <width>16777214</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>sendBox</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QTextEdit" name="m_outputEdit">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>50</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="m_sendButton">
                  <property name="text">
                   <string>send</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>高度距离</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="SliderV">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progBarV">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::BottomToTop</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
